pombola/south_africa/static/sass/_south-africa_overrides.scss

```diff
@@ -1173,6 +1173,9 @@ $logo-width-large: 184px;
                     line-height: 1.4;
                     letter-spacing: 0.01em;
                     width: 95%;
+                    @media screen and (max-width: 479px) {
+                        max-width: 80%;
```

Comment:
I don't think a %-based approach is correct - the thing on the right is fixed-width, which means we need to give it the space it needs, not a variable amount of space.

Reply:
noted, declining PR. I'll create another one.